README_Tiflow.md

```diff
@@ -93,11 +89,13 @@ If you have build the binary on your local Linux, you can try `../run-engine.sh
 
 ### Deploy
 
-There are several configure files to use. The file name suggests the number of running server-master and executor nodes. For example, `1m1e.yaml` means this file contains one server-master and one executor. Use `../run-engine.sh deploy 1m1e.yaml` to deploy cluster.
+There are several configure files to use. The file name suggests the number of running server-master and executor nodes.   
+For example, `1m1e.yaml` means this file contains one server-master and one executor.   
+Use `../run-engine.sh deploy ./deployments/engine/docker-compose/1m1e.yaml` to deploy cluster.
```

Comment:
Does this command work？

Reply:
![image](https://user-images.githubusercontent.com/17703670/186089863-2022c279-ed64-4f47-95d0-8f25404eaf59.png)
Sure. The original command fail after yujie move the `engine/deployment`  to `deployments/engine`. 
Just fix it.